controller/profiling.go

```diff
@@ -1,24 +1,21 @@
 package controller
 
 import (
-	"fmt"
 	"net/http"
 	"net/http/pprof"
 )
 
-const (
-	ProfilingPath = "/debug/pprof"
-)
-
 // NewPProfServer returns a new pprof server to gather runtime profiling data
 func NewPProfServer() *http.ServeMux {
 	mux := http.NewServeMux()
 
-	mux.HandleFunc(ProfilingPath, pprof.Index)
-	mux.HandleFunc(fmt.Sprintf("%s/cmdline", ProfilingPath), pprof.Cmdline)
-	mux.HandleFunc(fmt.Sprintf("%s/profile", ProfilingPath), pprof.Profile)
-	mux.HandleFunc(fmt.Sprintf("%s/symbol", ProfilingPath), pprof.Symbol)
-	mux.HandleFunc(fmt.Sprintf("%s/trace", ProfilingPath), pprof.Trace)
+	// TODO: Remove enumerating all pprof endpoints if/when a more ergonomic
+	// attachment solution is introduced. See: https://github.com/golang/go/issues/71213.
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
Note: I believe this is the source of the bug. `mux` needs the trailing slash to match requests properly:

https://github.com/golang/go/blob/46b576be724b6e64359fd872b9bd5109aba93cc0/src/net/http/pprof/pprof.go#L95-L105